zebra-chain/src/nullifier/sapling.rs

```diff
@@ -0,0 +1,18 @@
+use crate::serialization::ZcashSerialize;
+use serde::{Deserialize, Serialize};
+use std::io;
+
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Nullifier([u8; 32]);
+
+impl From<[u8; 32]> for Nullifier {
+    fn from(buf: [u8; 32]) -> Self {
+        Self(buf)
+    }
+}
+
+impl ZcashSerialize for Nullifier {
```

Comment:
Also `ZcashDeserialize`

Reply:
@hdevalence do we need to impl `Zcash(De)Serialize` for nullifiers? My instinct is yes. If yes, we need to use `zcash_serialize` and `zcash_deserialize` in the respective places [below](https://github.com/ZcashFoundation/zebra/pull/568/files#diff-dc2aa89a91bbadeda1b2a78ce306eb99R330), not `From/Into`.